tests/candev/Makefile

```diff
@@ -7,6 +7,15 @@ USEMODULE += isrpipe
 # define the CAN driver you want to use here
 CAN_DRIVER ?= native
 
+# prevent using native driver on non-native board
+ifeq ($(CAN_DRIVER), native)
+ifneq ($(BOARD), native)
+$(error native can driver can only be used on native board!)
+endif
+endif
+
+
+
 ifeq ($(CAN_DRIVER), PERIPH_CAN)
 # periph_can modules/variables go here
 
```

Comment:
Question: the check bellow is supposed to match the default `native` value of `CAN_DRIVER`?

Reply:
I was confused about what you were asking here at first, but then it dawned on me: we set the default value to `native`, but check for a set define called `CAN_NATIVE`. Good catch! 

I suggest changing the default value though, so it follows the convention.